### PR TITLE
Update of PT-PT translation

### PR DESCRIPTION
**Pull request by MalMen**

### CODE REVIEW

**Comment by erciccione** [comment]
ref #1377

File: translations/monero-core_pt-pt.ts [hunk 1]
```diff
@@ -2510,65 +2510,70 @@ Custo: </translation>
...
     </message>
     <message>
         <location filename="../main.qml" line="627"/>
         <source>
 
 WARNING: non default ring size, which may harm your privacy. Default of 7 is recommended.</source>
-        <translation type="unfinished"></translation>
+        <translation>
+
+ATENÇÃO: um tamanho de ring que não o de defeito pode enfraquecer a sua privacidade. O ring de 7 é o recomendado.</source></translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: remove `</source>`

File: translations/monero-core_pt-pt.ts [hunk 2]
```diff
@@ -1221,155 +1221,155 @@ O ficheiro cache antigo da carteira será renomeado e poderá ser reposto depois
...
         <location filename="../pages/SharedRingDB.qml" line="206"/>
         <source>Paste output public key</source>
-        <translation type="unfinished"></translation>
+        <translation>Cole o endereço publico</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="220"/>
         <source>Blackball</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Adicionar</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: is this left 'unfinished' on purpose?

File: translations/monero-core_pt-pt.ts [hunk 3]
```diff
@@ -1221,155 +1221,155 @@ O ficheiro cache antigo da carteira será renomeado e poderá ser reposto depois
...
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="229"/>
         <source>Unblackball</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Remover</translation>
```
Comment (erciccione) [inline translations/monero-core_pt-pt.ts]: same as above

**Comment by erciccione** [comment]
ping @MalMen

File: translations/monero-core_pt-pt.ts (hunk 2 above)
Comment (MalMen) [reply translations/monero-core_pt-pt.ts]: Yes, I am not sure if this translation is the correct aproach, I need to see where this stay on the GUI to be sure

File: translations/monero-core_pt-pt.ts (hunk 3 above)
Comment (MalMen) [reply translations/monero-core_pt-pt.ts]: same

File: translations/monero-core_pt-pt.ts (hunk 2 above)
Comment (erciccione) [reply translations/monero-core_pt-pt.ts]: Build is broken right now, you should cherrypick monero-project/monero#3745 to be able to compile. Btw that "blackball" is basically "blacklist". Check [this section of the guide](https://github.com/monero-ecosystem/monero-GUI-guide/blob/master/monero-GUI-guide.md#53-shared-ringdb) to see where those buttons are

**Pull request merged**